frontend/src/premium/user/components/list/user-list-table.vue

```diff
@@ -14,7 +14,7 @@
       <span class="leading-5">Invite user</span>
     </el-button>
   </div>
-  <div class="app-list-table panel">
+  <div class="user-list-table panel">
```

Comment:
The idea with `app-list-table` was to have a generic CSS class.
Can we not use it on this `user-list-table.vue` component?

Reply:
Thanks for the heads up. I didn't remove it, I used the `app-list-table` class but added an extra class to prevent the style I wanted.

I added `non-clickable` class; if present, the row on hover won't have the cursor pointer